### PR TITLE
Update the doc of DivOverlay

### PR DESCRIPTION
**Pull request by Falke-Design**
Fix: #6597

- DivOverlay has a default offset: [0,7] I think this would be better if this is moved to Popup and changed to [0,0] but this is a breaking change
- DivOverlay has not factory function

### CODE REVIEW

File: src/layer/DivOverlay.js [hunk 1]
```diff
@@ -19,16 +19,16 @@ export var DivOverlay = Layer.extend({
 	// @aka DivOverlay options
 	options: {
 		// @option offset: Point = Point(0, 7)
-		// The offset of the popup position. Useful to control the anchor
+		// The offset of the overlay position. Useful to control the anchor
 		// of the popup when opening it on some overlays.
```
Comment (johnd0e) [inline src/layer/DivOverlay.js:23]: > when opening it on some overlays

What does it ever mean?

File: src/layer/DivOverlay.js [hunk 2]
```diff
@@ -19,16 +19,16 @@ export var DivOverlay = Layer.extend({
...
 
 		// @option className: String = ''
-		// A custom CSS class name to assign to the popup.
+		// A custom CSS class name to assign to the overlay.
 		className: '',
 
 		// @option pane: String = 'popupPane'
-		// `Map pane` where the popup will be added.
+		// `Map pane` where the overlay will be added.
 		pane: 'popupPane'
```
Comment (johnd0e) [inline src/layer/DivOverlay.js:32]: Shouldn't we change option default value? May be unset?

File: src/layer/DivOverlay.js [hunk 3]
```diff
@@ -214,6 +215,11 @@ export var DivOverlay = Layer.extend({
 			}
 			node.appendChild(content);
 		}
+
+		// @namespace DivOverlay
+		// @section DivOverlay events
+		// @event contentupdate: Event
+		// Fired when the content of the overlay is updated
 		this.fire('contentupdate');
```
Comment (johnd0e) [inline src/layer/DivOverlay.js:223]: Interesting..

**Comment by johnd0e** [comment]
> * DivOverlay has a default offset: [0,7] I think this would be better if this is moved to Popup and changed to [0,0] but this is a breaking change

Hm.. Still I'd prefer to make that change.

> * DivOverlay has not factory function

And that is right, because it is abstract class.

File: src/layer/DivOverlay.js (hunk 1 above)
Comment (Falke-Design) [reply src/layer/DivOverlay.js:23]: I don't know. Maybe we can remove the second sentence

File: src/layer/DivOverlay.js (hunk 2 above)
Comment (Falke-Design) [reply src/layer/DivOverlay.js:32]: if it is a abstract class (like you said), then I would say yes

**Comment by johnd0e** [comment]
@Falke-Design

**Review by johnd0e — APPROVED** [review]

**Comment by Falke-Design** [comment]
To keep it clear before merging: This is a breaking change! The default values for DivOverlay are chaged

**Review by mourner — APPROVED** [review]

**Pull request merged**